src/test/java/org/reactome/release/orthopairs/UniProtGeneNamesRetrieverTest.java

```diff
@@ -0,0 +1,78 @@
+package org.reactome.release.orthopairs;
+
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+public class UniProtGeneNamesRetrieverTest {
+
+    @Test
+    public void testGetUniProtIdentifiers() {
+
+        Set<String> testProteinIds = new HashSet<>(Arrays.asList("UniProtKB=12345"));
+        Map<String, Set<String>> fakePantherMappings = new HashMap<>();
+        fakePantherMappings.put("testKey", testProteinIds);
+
+        Set<String> testUniProtIds = UniProtGeneNamesRetriever.getUniProtIdentifiers(fakePantherMappings);
+
+        assertThat(testUniProtIds, hasSize(1));
+        assertThat(testUniProtIds.contains("12345"), is(equalTo(true)));
```

Comment:
This could be `assertThat(testUniProtIds, hasItem("12345"));` or `assertThat(testUniProtIds, contains("12345"))` since there is only one element.